### ISSUE TITLE
Open Project, Do Nothing, Write Project - Corrupts Project

### ISSUE THREAD

**Issue by idi888** [tuist/xcodeproj#346]
I was working on a simply program to adding a dependancy to target project, and my project keep getting corrupted. 

So I just tried to simply open, write  and I got the same error
What am I doing wrong?

```            
let path = Path(file.location.path)
let project = try XcodeProj(path: path)
try project.write(path: path)
```
![screen shot 2018-11-09 at 4 18 13 pm](https://user-images.githubusercontent.com/8197402/48244801-95122d80-e43b-11e8-8224-15d57c7f9158.png)

**Comment by pepibumur** [comment]
Hey @idi888, thanks for reporting this. Could you share the `git diff` after executing those three lines? That would help us understand what might have got corrupted in your project, and see if there's anything we need to fix in xcodeproj.

**Comment by pepibumur** [comment]
This should be fixed in the latest version of the library, 6.7.0. If it keeps happening, feel free to open a new issue.
